src/pudl/metadata/resources/ferceqr.py

```diff
@@ -0,0 +1,177 @@
+"""Table definitions for the FERC EQR data group."""
+
+from typing import Any
+
+TABLE_DESCRIPTIONS = {
+    "identity": {
+        "additional_summary_text": (
+            "individuals who filed FERC EQR for a company in a given quarter."
+        ),
+        "additional_primary_key_text": (
+            "The primary key ought to be ['year_quarter', 'company_id_ferc', "
+            "'filer_unique_id'], where filer_unique_id is an employee-level ID. "
+            "However, a handful of companies have erroneously reported the same "
+            "filer_unique_id for multiple employees, resulting in duplicate records. "
+            "In other cases, there appear to be multiple filings in a given quarter "
+            "for the same company and filer, resulting in additional duplicates. Thus, "
+            "there is no reliable natural primary key for the identity table."
+        ),
+        "usage_warnings": ["experimental_wip"],
+    },
+    "contracts": {
+        "additional_summary_text": (
+            "Contains information about contracts between power selling and"
+            " purchasing companies."
```

Comment:
Generalize since it's not just power (energy).
```suggestion
            "Contains information about contracts between companies selling"
            " and buying electricity market products."
```